src/Dotnet.Script.DependencyModel/Dotnet.Script.DependencyModel.csproj

```diff
@@ -32,7 +32,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NuGet.ProjectModel" Version="5.2.0" />
```

Comment:
omnisharp is still on `6.3.0-rc.128`, let's not use a higher version yet, so I'd suggest 6.2?